src/workloads/query/query_stats/BaseFixed.yml

```diff
@@ -0,0 +1,41 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This workload tests for performance regressions without concurrent $queryStats aggregates,
+  using fixed-size payloads in the find queries.
```

Comment:
fixed size of variable shape or all of the same shape? Or cycling? Or all of the above?